public/quotes.js

```diff
@@ -154,5 +154,9 @@ var quotes = [
   {
     quote: "We have developers who kind of develop",
     author: "Andreas"
+  },
+  {
+    quote: "...(typing)...",
+    author: "Andreas"
   }
```

Comment:
Is it possible to add a `,` in the end here so in the future we only se the object added not an extra line in the top?